javascript/javascript1/week4/task_1.js

```diff
@@ -0,0 +1,86 @@
+let myName = "";
+let todo = [];
+
+function getReply(command) {
+  const array = command.split(" ");
+  let response="";
+  if (command.includes("hello my name is")) {
+    myName = array[4];
+    response = `nice to see you ${myName}`;
+  } else if (command.includes("what is my name")) {
+    response = `my name is ${myName}`;
+  } else if (command.includes("add fishing")) {
+    todo.push("fishing");
+    response = "added fishing to todo";
+  } else if (command.includes("add singing")) {
+    todo.push("singing");
```

Comment:
See, there could be 100 same commands, even some unknown tasks like `add travelling to new places` then it won't work, right?